importer/api/__init__.py

```diff
@@ -124,6 +125,7 @@ def import_course(bundle, repo_id, user_id, static_dir):
     )
     import_static_assets(course, static_dir)
     import_children(course, src, None)
+    populate_xanalytics_fields.delay(course.id)
```

Comment:
Yes, because even if we trap all "expected" errors when accessing an external API (such as timeouts), it could fail in a new and wonderful way. That shouldn't break the course import.